.github/label-actions.yml

```diff
@@ -113,7 +113,7 @@
     Hi there,
 
 
-    Please give us specific logs, so we can help you more.
+    A maintainer would like for you to locate and share a specific log message (`packageFiles with updates`) from your repo/environment.
```

Comment:
No, this is a wrong change. I'm trying to avoid the confusion of the current "we're interested in some logs" and replace it with "there's a very specific log message we need"

Reply:
```suggestion
    A maintainer wants you to find and share a specific log message (`packageFiles with updates`) from your repository or environment.
```

Let's keep the format, but use simpler words. :wink: